src/main/java/com/thebois/views/ColonyView.java

```diff
@@ -0,0 +1,39 @@
+package com.thebois.views;
+
+import java.util.Collection;
+
+import com.badlogic.gdx.graphics.Color;
+import com.badlogic.gdx.graphics.glutils.ShapeRenderer;
+
+import com.thebois.models.beings.IBeing;
+
+/**
+ * Used to display all pawns.
+ */
+public class ColonyView {
```

Comment:
You could make it more general: `BeingView`.

Reply:
Will fix this.